frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -1,18 +1,29 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import React, { useState } from 'react';
+import React, { useState, useContext } from 'react';
+import { DECIMAL_BASE } from '@ttahub/common';
 import { v4 as uuidv4 } from 'uuid';
 import { useHistory } from 'react-router-dom';
+import UserContext from '../../../UserContext';
 import './EventCard.scss';
 import { eventPropTypes } from '../constants';
 import TooltipList from '../../../components/TooltipList';
 import ContextMenu from '../../../components/ContextMenu';
 import { checkForDate } from '../../../utils';
 import ExpanderButton from '../../../components/ExpanderButton';
 import SessionCard from './SessionCard';
+import { canEditOrCreateSessionReports } from '../../../permissions';
 
 function EventCard({
   event,
 }) {
+  const { user } = useContext(UserContext);
+  const hasEditPermissions = canEditOrCreateSessionReports(
+    user,
+    parseInt(event.regionId, DECIMAL_BASE),
+  );
+  const isCollaborator = event.collaboratorIds.includes(user.id);

```

Comment:
```suggestion
  const isCollaborator = event.pocId.includes(user.id);

```

Reply:
I know its a little weird, but collaboratorIds is "IST Collaborators" and "pocId" is "Regional Point of Contact Collaborators"

Reply:
mostly unrelated to this PR: (sorry to hijack)

I didn't realize `pocId` was migrated to an array. I think the tests are all still written to validate it as a `number`.

come to think of it, the `createEvent` handler just takes whatever you give it for the value and sticks it in the db, but this test is still passing??

https://github.com/HHS/Head-Start-TTADP/blob/f18cf62fdd84b81c107b84dac7bedd1b3a95da83/src/routes/events/handlers.test.js#L125-L145

Reply:
I updated the services test, but I guess I missed this one! 

Looks like the only validation that is happening is whether or not pocId is truthy, unless I'm missing something somewhere?

Reply:
as would I